scene/3d/bone_attachment_3d.cpp

```diff
@@ -57,7 +57,7 @@ bool BoneAttachment3D::_set(const StringName &p_path, const Variant &p_value) {
 	if (p_path == SNAME("use_external_skeleton")) {
 		set_use_external_skeleton(p_value);
 	} else if (p_path == SNAME("external_skeleton")) {
```

Comment:
```suggestion
#ifndef DISABLE_DEPRECATED
	} else if (p_path == SNAME("external_skeleton")) {
		r_ret = get_external_skeleton();
#endif // DISABLE_DEPRECATED
```
You can use the `DISABLE_DEPRECATED` block to migrate while changing property names, as some other classes have done. Then, the property name must also be changed in `_set()` and `_get_property_list()` as well.